@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -640,30 +619,29 @@ export class TextElementsRenderer {
                     if (pathBlockingElement.points.length < 2) {
                         continue;
                     }
-                    // Project to screen and store it in the ScreenCollisions.
                     let startLinePoint = pathBlockingElement.points[0];
                     for (let i = 1; i < pathBlockingElement.points.length; i++) {
                         const endLinePoint = pathBlockingElement.points[i];
-                        this.m_screenProjector.projectAndShift(startLinePoint, startLinePointProj);
-                        this.m_screenProjector.projectAndShift(endLinePoint, endLinePointProj);
+                        this.m_screenProjector.project3(startLinePoint, startLinePointProj);
+                        this.m_screenProjector.project3(endLinePoint, endLinePointProj);
```

Comment:
Can we not save the projected start point together with the unprojected "startLinePoint" to save one projection?

Reply:
yes, good point.